lib/profile/view_profile/sections/education.dart

```diff
@@ -1,27 +1,33 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
 
 import '../../../constants/profile_constants.dart';
+import '../../../models/models.dart';
+import '../../../selectors/selectors.dart';
 import '../../common/school_row.dart';
 import '../../common/section.dart';
 
  
 class ProfileEducation extends StatelessWidget {  
-  final List<Map<String, Object>> schools;
-  ProfileEducation({Key key, @required this.schools}) : super(key: key);
+  ProfileEducation({Key key}) : super(key: key);
   
   @override
   Widget build(BuildContext context) {
-    return Section(
-      title: sectionTitleEducation,
-      child: buildSchoolList(context)
+    return StoreConnector<AppState, List<School>>(
+      converter: schoolListSelector,
+      builder: (context, schools) => schools != null 
+      ? Section(
+        title: sectionTitleEducation,
+        child: buildSchoolList(schools)
+      ) : SizedBox()
```

Comment:
BTW, I made it so that the sections (Summary, Education, Experience, Skills) won't be visible in the ViewProfile page if they don't have any content in them

Reply:
Yep, this is what the sections in EditProfile will look like when they're empty:

<img width="300" alt="Screen Shot 2020-07-22 at 4 27 26 PM" src="https://user-images.githubusercontent.com/25041263/88235508-3e14b200-cc38-11ea-9186-cdced7ccf997.png">